integration_tests/sdk/aqueduct_tests/integration_test.py

```diff
@@ -46,3 +50,11 @@ def test_invalid_connect_integration(client):
     # Invalid config raises a pydantic error.
     with pytest.raises(ValidationError):
         client.connect_integration("New Integration", "SQLite", {})
+
+
+@pytest.mark.enable_only_for_engine_type(ServiceType.K8S)
+def test_sqlite_with_k8s(data_integration, engine):
+    """Tests that running an extract operator that reads data from a SQLite database using k8s should fail."""
+    global_config({"engine": engine})
+    with pytest.raises(AqueductError):
```

Comment:
can you use the "match=<err msg>" argument here to make it clear what the expected error message is?